src/mockData/mockData.js

```diff
@@ -1,5 +1,6 @@
 export const lesson = {
   name: 'Test Title',
+  id: 123123,
   questions: [
```

Comment:
This was a cause of some hurdles, but I cleared them!

Reply:
That would do it!  Nice job finding that.